### ISSUE TITLE
problem with point in table or steam name

### ISSUE THREAD

**Issue by tkaven** [confluentinc/ksql#2172]
```
ksql> SELECT * FROM `vs-hgame-projects-n1_stream` limit 3;
Invalid Expression vs-hgame-projects-n1_stream.ROWTIME.
```

**Comment by rmoff** [comment]
@tkaven what version are you using? 

There are some known issues with hyphens in object names, including #1165, #769 , #1466

**Comment by tkaven** [comment]
> @tkaven what version are you using?
> 
> There are some known issues with hyphens in object names, including #1165, #769 , #1466
my version is 5.0.1
thank you

**Comment by miguno** [comment]
This should be a duplicate of https://github.com/confluentinc/ksql/issues/2176 (also reported by @tkaven).

**Comment by miguno** [comment]
And this issue here is also related to https://github.com/confluentinc/ksql/issues/1466, aka "KSQL should support object names with the - character (hyphen, dash) #1466".
